tests/test_ensembles/test_stacking_ensemble.py

```diff
@@ -252,7 +252,7 @@ def test_forecast_calls_process_forecasts(example_tsds: TSDataset, naive_ensembl
     naive_ensemble.fit(ts=example_tsds)
     naive_ensemble._process_forecasts = MagicMock()
 
-    result = naive_ensemble._forecast()
+    result = naive_ensemble._forecast(return_components=False)
```

Comment:
Why should we add `return_components=False` here? Is it because we don't have default value here?

Reply:
Yes, as `return_components` should be passed from the public method, otherwise we might forget to pass it and everything will still work